ui/components/common/Validators.js

```diff
@@ -1,3 +1,4 @@
+const MAX_PORT_NUM = 65535;
```

Comment:
Isn't this #68 's ? :P

Reply:
Yes but i thought why not just do it all in one

Reply:
@camilamacedo86 is that ok for you?

Reply:
👍perfect..  Just close that one.

Reply:
@pb82, It is fine .. However, maybe something, as follows, could make clear what it is doing. 

```javascript 

/**
 * Run a number of validations that have to be passed in an array of the form:
 * [
 *  <Validator 1>, <Input Value 1>
 *  ...
 *  <Validator N>, <Input Value N>
 * ]
 * @param validations The validations array
 * @returns {string} "success" on success, otherwise "error"
 */
const Validate = validations => {
    // It always has to be exactly one validator per input value, so the total length
    // of the array mod 2 must be zero.
    function hasQtExpectedPositions() {
        return validations && validations.length % 2 !== 0;
    }
    function hasInputValue(arr, index) {
        return arr[index](arr[index + 1]);
    }
    // For each validator is expected an input value
    function hasExpectedValues(arr) {
        for (let i = 0; i < arr.length; i += 2) {
            if (!hasInputValue(arr, i)) {
                return false;
            }
        }
        return true;
    }
    if (!hasQtExpectedPositions() || !hasExpectedValues(validations)) {
        return "error";
    }
    return "success";
};

export { Validators, Validate };


```

Reply:
@camilamacedo86 maybe this example makes it clear how it is used: https://github.com/aerogear/data-sync-ui/blob/master/ui/components/data-sources/BaseDataSourceDialog.js#L66